course/include/Vector3.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+#include <iostream>
+#include "math.h"
+
+namespace ekumen
+{
+
+class Vector3
+{
+public:
+    explicit Vector3();
+    explicit Vector3(const double x, const double y, const double z);    
+    Vector3(const std::initializer_list<double> &list);
+
+    double x() const {return values[0];}
```

Comment:
nit: This is fine, but you could instead return a `const double &`, which will in general avoid an additional copy. For `double` is not really important, but for larger objects it may be.